test/runtests.jl

```diff
@@ -33,6 +33,7 @@ if GROUP == "All" || GROUP == "Interface"
     @time @safetestset "Upwind Operator Interface" begin include("upwind_operators_interface.jl") end
     @time @safetestset "MOLFiniteDifference Interface" begin include("MOLtest.jl") end
     @time @safetestset "Basic SDO Examples" begin include("BasicSDOExamples.jl") end
+    @time @safetestset "3D laplacian Test" begin include("3D_laplacian.jl") end
```

Comment:
Something's weird -- I'm not sure this is from the right commit.  Line 23 of test/runtests.jl is causing an error that's fixed post-rebase (cf. https://github.com/xtalax/DiffEqOperators.jl/pull/26/files#diff-fce720c43af3c52c862fd7451c7374b8)